go.mod

```diff
@@ -7,48 +7,48 @@ require (
 	github.com/astaxie/beego v1.12.0
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/container-storage-interface/spec v1.5.0
-	github.com/distribution/distribution/v3 v3.0.0-20220526142353-ffbd94cbe269
-	github.com/docker/docker v20.10.17+incompatible
+	github.com/distribution/distribution/v3 v3.0.0-20221208165359-362910506bc2
+	github.com/docker/docker v20.10.21+incompatible
 	github.com/docker/go-connections v0.4.0
 	github.com/eclipse/paho.mqtt.golang v1.2.0
 	github.com/emicklei/go-restful v2.9.6+incompatible
-	github.com/evanphx/json-patch v4.12.0+incompatible
+	github.com/evanphx/json-patch v5.6.0+incompatible
 	github.com/golang-jwt/jwt v3.2.2+incompatible
 	github.com/golang/mock v1.6.0
-	github.com/golang/protobuf v1.5.2
-	github.com/google/uuid v1.2.0
+	github.com/golang/protobuf v1.5.3
+	github.com/google/uuid v1.3.0
 	github.com/gorilla/websocket v1.4.2
 	github.com/kubeedge/beehive v0.0.0
 	github.com/kubeedge/viaduct v0.0.0
 	github.com/kubernetes-csi/csi-lib-utils v0.6.1
-	github.com/mattn/go-sqlite3 v1.14.9
+	github.com/mattn/go-sqlite3 v1.14.14
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.19.0
-	github.com/prometheus/client_golang v1.12.1
+	github.com/prometheus/client_golang v1.14.0
 	github.com/shirou/gopsutil v2.21.11+incompatible
 	github.com/shirou/gopsutil/v3 v3.23.2
-	github.com/spf13/cobra v1.4.0
+	github.com/spf13/cobra v1.7.0
 	github.com/spf13/pflag v1.0.5
 	golang.org/x/net v0.8.0
 	golang.org/x/time v0.0.0-20220210224613-90d013bbcef8
-	google.golang.org/grpc v1.43.0
-	google.golang.org/protobuf v1.27.1
-	helm.sh/helm/v3 v3.9.4
-	k8s.io/api v0.24.14
-	k8s.io/apiextensions-apiserver v0.24.14
-	k8s.io/apimachinery v0.24.14
-	k8s.io/apiserver v0.24.14
-	k8s.io/cli-runtime v0.24.14
-	k8s.io/client-go v0.24.14
```

Comment:
The kubernetes related versions also need to update?

Reply:
And it's better to split go mod in one commit, others in another :)

Reply:
> The kubernetes related versions also need to update?

 Kubernetes versions are updated when bump helm to 3.11.1, but these are replaced to v1.24.4-kubeedge1 in the back part.

Reply:
> And it's better to split go mod in one commit, others in another :)

done.